codebase2/codebase-sqlite/U/Codebase/Sqlite/Queries.hs

```diff
@@ -268,11 +274,16 @@ createSchema :: Transaction ()
 createSchema = do
   executeFile [hereFile|unison/sql/create.sql|]
   addTempEntityTables
+  addNamespaceStatsTables
 
 addTempEntityTables :: Transaction ()
 addTempEntityTables =
   executeFile [hereFile|unison/sql/001-temp-entity-tables.sql|]
 
+addNamespaceStatsTables :: Transaction ()
+addNamespaceStatsTables =
+  executeFile [hereFile|unison/sql/003-namespace-statistics.sql|]
```

Comment:
I'm pretty sure the reflog one will be merged first, but reminder for myself to swap this to `002` if not haha.

Reply:
Oops yeah I just merged the reflog one, and then started worrying about this as I was scrolling through this PR